posts/rise-of-microservices-and-docker.md

```diff
@@ -0,0 +1,93 @@
+# Rise of Microservices and Docker
+
+Are you a software engineer/developer/coder or maybe even a tech enthusiast who hears terms like `Microservices` and `Docker`? 
+
+Are you someone who has been encountering these terminologies so often through various channels like articles, news, blogs, meetups, conferences but still clueless there's so much hype?
+
+If yes, this is a must-read for you!
+
+## A little bit of "History"
```

Comment:
To be honest, I don't think you need to include this piece about SOA.

Reply:
Hey Sid, I feel this is really important because there has been multiple discussions around it among DevOps and this definitely gives them a big picture of evolution of software architecture rather than just getting to use the technologies. It helps them appreciate why there has been changes in the trends and who or what factors decide them